templates/category/index.html

```diff
@@ -1,30 +1,145 @@
+                 data-icon-down="{% static "assets/chevron_down.svg" %}">
+              <form method="get">
+                {% for field in filter.form %}
+                  {% if field.name == 'sort_by' %}
+                    <!--Field 'sort_by' is hidden because it is rendered in header.
+                    This is required if you want to have sorting to be kept during further filtering.-->
+                    <input type="hidden" name="sort_by"
+                           value="{% if request.GET.sort_by %}{{ request.GET.sort_by }}{% endif %}">
+                  {% elif field.name != 'price' %}
+                    <div class="toggle-filter">
+                      <h3 class="filter-label">
+                        {{ field.label }}
+                        <div class="collapse-filters-icon">
+                          <img src="{% static "assets/chevron_up.svg" %}">
+                        </div>
+                      </h3>
+                      <div class="filter-form-field" style="display:block">
+                        {{ field }}
+                      </div>
+                    </div>
+                  {% endif %}
+                {% endfor %}
+                {% for field in filter.form %}
+                  {% if field.name == 'price' %}
+                    <div class="toggle-filter product-filters__price-range">
+                      <h3 class="filter-label">
+                        {{ field.label }}
+                        <div class="collapse-filters-icon">
+                          <img src="{% static "assets/chevron_up.svg" %}">
+                        </div>
+                      </h3>
+                      <div class="filter-form-field" style="display:block">
+                        <div>
+                          <input id="{{ field.auto_id }}_0" name="{{ field.name }}_0"
+                                 value="{% if field.value.0 %}{{ field.value.0 }}{% endif %}" type="number" min="0"
+                                 class="form-control d-inline"
+                                 placeholder="from"><span>-</span><input id="{{ field.auto_id }}_1" name="{{ field.name }}_1"
+                                 value="{% if field.value.1 %}{{ field.value.1 }}{% endif %}" type="number" min="0"
+                                 class="form-control d-inline" placeholder="to">
+                        </div>
+                      </div>
+                      <button class="btn primary" type="submit">Update</button>
+                    </div>
+                  {% endif %}
+                {% endfor %}
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col-md-8 col-lg-9 category-list">
+        <div>
+          <div>
+            <div class="row">
+              {% include "category/_items.html" with products=products %}
+            </div>
+            <div class="row">
+              <div class="m-auto">
+                {% if show_pagination %}
```

Comment:
`{% if products_paginated.has_other_pages %}` is much better way of checking this and doesn't make any queries.

Reply:
Done